manifests/h/Hashicorp/Vagrant/2.3.3/Hashicorp.Vagrant.installer.yaml

```
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Hashicorp.Vagrant
+PackageVersion: 2.3.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+- 1706
+UpgradeBehavior: install
+InstallLocationRequired: true
+RequireExplicitUpgrade: true
+DisplayInstallWarnings: false
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://releases.hashicorp.com/vagrant/2.3.3/vagrant_2.3.3_windows_amd64.msi
+  InstallerSha256: DE268C1D580066E23F6E334E3EA14850CD4AEB186E545AF08B1F7217FF67E6E7
+  ProductCode: '{608AAE8F-FAA0-4181-AB93-EA9AAEE0ED6E}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://releases.hashicorp.com/vagrant/2.3.3/vagrant_2.3.3_windows_i686.msi
+  InstallerSha256: 2C9AB69BEE74CF19894B40BC7E2EC30B3C00DD9B425C611EB53987D900FC0D5D
+  ProductCode: '{9423FCE4-520A-434F-8771-B373503E44DA}'
+ManifestType: installer
+ManifestVersion: 1.2.0
+ReleaseDate: 2022-08-05

```

```suggestion
ReleaseDate: 2022-11-15
```
https://github.com/hashicorp/vagrant/blob/v2.3.3/CHANGELOG.md shows version was released on `November 15, 2022`.